sam/README.md

```diff
@@ -0,0 +1,15 @@
+# Sam
+
+This module enables the APIs and creates the service accounts needed to run the
+[Sam service](https://github.com/broadinstitute/sam/).
+
+It does not create the persistent storage resources that Sam relies on. We plan on sharing the persistent storage from the "classic" Terra deployment, so we need to distinguish between
+resources to be shared with the classic deployment and resources that are independent to new deployments.
+
+Resources in this folder are created by the classic deployment today in dev/staging/production "shared" environments.
+To create the same resources for not-shared environments exclusive to this deployment, we need to be create the same
+resources. This module handles those resources.
+
+
+## TODO
+Need to add manual steps for GSuite service accounts. See terraform-terra repo.
```

Comment:
IMO we should try and keep track of manual steps in some standardized, organized way. Need to think about the best approach but maybe a README in the module root for now? Could probably circle back and add this after we actually get SAM up and running so we know for sure what all the manual steps are.

Reply:
We should definitely keep track of the steps. I'm open to any approach, whether that's a manual_steps.py or a readme section.

Reply:
To be clear, I haven't done the manual steps yet, so I don't know the full extent. I'm planning on adding them in follow up PRs.